### PR TITLE
Hide tokens

### LINKED ISSUE #1189: Hide tokens in preference menu
Preferences -> Security

- [ ] Make sure that all sensitive issue is hidden
- [ ] Review that no other data is visible. 

Imagine that you are doing a presentation and everybody is looking and recording your screen. Ask yourself: what data should be hidden??

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?
GUI doesn't show token secret and key.
image

## Related issue number
related to #1189

### CODE REVIEW

File: services/web/client/source/class/osparc/desktop/preferences/pages/SecurityPage.js [hunk 1]
```diff
@@ -122,33 +121,52 @@ qx.Class.define("osparc.desktop.preferences.pages.SecurityPage", {
       return form;
     },
 
-    __createValidTokenForm: function(service, key, secret) {
-      let form = new qx.ui.form.Form();
+    __createValidTokenForm: function(token) {
+      const service = token["service"];
```
Comment (pcrespov) [inline services/web/client/source/class/osparc/desktop/preferences/pages/SecurityPage.js]: The tokens, once transmitted ... MUST NOT be stored anywhere on the client side

Comment (odeimaiz) [reply services/web/client/source/class/osparc/desktop/preferences/pages/SecurityPage.js]: I know, this PR is not closing the tokens issue, is just about not showing them.

In the same direction, when the frontend asks for the tokens, the backend should only answer with a list of service names.

Comment (pcrespov) [reply services/web/client/source/class/osparc/desktop/preferences/pages/SecurityPage.js]: ok then...make sure to keep open then #1189

File: services/web/client/source/class/osparc/desktop/preferences/pages/SecurityPage.js [hunk 2]
```diff
@@ -122,33 +121,49 @@ qx.Class.define("osparc.desktop.preferences.pages.SecurityPage", {
...
+        column: 1
       });
-      form.add(tokenService, this.tr("Service API"));
 
-      let tokenKey = new qx.ui.form.TextField();
-      tokenKey.set({
-        value: key,
-        readOnly: true
+      /*
+      const showTokenIcon = "@FontAwesome5Solid/edit/"+iconHeight;
```
Comment (ignapas) [inline services/web/client/source/class/osparc/desktop/preferences/pages/SecurityPage.js]: Commented code can go?

Comment (odeimaiz) [reply services/web/client/source/class/osparc/desktop/preferences/pages/SecurityPage.js]: Ready for when the backend provides such a feature

**Pull request merged**